frontend/src/components/game/Editor.tsx

```diff
@@ -98,13 +127,29 @@ function ResizableMonacoEditor(props: EditorProps) {
       <EditorContainer>
         <MonacoEditor
           options={{
+            automaticLayout: true,
             fixedOverflowWidgets: true,
+            fontFamily: 'Monaco',
+            hideCursorInOverviewRuler: true,
             minimap: { enabled: false },
-            automaticLayout: true,
+            overviewRulerBorder: false,
+            overviewRulerLanes: 0,
+            padding: { top: 10, bottom: 10 },
+            quickSuggestions: false,
+            renderLineHighlight: 'all',
+            renderIndentGuides: false,
+            renderWhitespace: 'none',
+            scrollbar: {
+              verticalScrollbarSize: 5,
+              horizontalScrollbarSize: 5,
+              useShadows: false,
+            },
             scrollBeyondLastLine: false,
           }}
```

Comment:
nit: Define the options in a separate variable and put that variable here.

Reply:
Done - will note that I had to add a type any to the separate variable in order to get it to be compatible with this options parameter 

I think another comment mentioned the use of any so I can look into if types exist when going over that one

Reply:
No problem at all, yeah I think the specific types only apply when it's reasonably easy, for some of the external libraries it's tough to find the types.